src/content/convert/convert-nodes.ts

```diff
@@ -13,6 +14,7 @@ export const convertNode: SConvertNode = async (state, target, nodes) => {
         .then(() => convertNodesText(target, parsedNodes))
         .then(texts => {
           state.mutationObserver?.disconnect();
+          state.updateLang && updateHtmlLang(document.documentElement, target);
```

Comment:
please swap line 17 and 18.
(make sure the lang attribute is only set when the content is successfully updated)

Reply:
Updated in 44841ddb49191f607bc8a5d2e9018a7e1f67a16a